BMM.Core/Implementations/PlayObserver/Storage/TrackPlayedStorage.cs

```diff
@@ -11,7 +11,7 @@ namespace BMM.Core.Implementations.PlayObserver.Storage
 {
     public class TrackPlayedStorage : ITrackPlayedStorage
     {
-        private const string StorageKey = StorageKeys.FinishedTrackPlayedEvents;
+        private readonly string _storageKey = StorageKeys.FinishedTrackPlayedEvents;
```

Comment:
why did you change this?
Can you at least make it static so that we can keep the PascalCase naming and don't need the underscore?

Reply:
doesn't really matter